charts/console/README.md

```diff
@@ -0,0 +1,13 @@
+# Redpanda Console Helm Chart
+
+This Helm chart allows you to deploy Redpanda Console to your Redpanda cluster.
+You can install the chart by running the following commands:
+
+```shell
+helm repo add redpanda 'https://charts.redpanda.com/' 
+helm repo update
+helm install redpanda/console -f myvalues.yaml
+```
+
+Have a look at the [values.yaml](./console/values.yaml) file to see the available options.
```

Comment:
The path to the values.yaml is wrong I believe. Should be `./values.yaml` I believe